README.md

```diff
@@ -0,0 +1,29 @@
+# pulseaudio
+
+This role installs [PulseAudio](https://www.freedesktop.org/wiki/Software/PulseAudio/) and creates a
+[systemd](https://wiki.debian.org/systemd) service called `pulseaudio.service` in order to start
+PulseAudio automatically.
+If PulseAudio is already ran by the same user (see [`pulseaudio_user`](#role-variables)), then our
+service will kill that instance prior to starting its own.
+
+## Requirements
+
+An apt based system like [Ubuntu](https://www.ubuntu.com/) or [Debian](https://www.debian.org/).
+
+## Role Variables
+
+| Name              |   Default    | Description                           |
+| :---------------- | :----------: | :------------------------------------ |
+| `pulseaudio_user` | `pulseaudio` | The user running `pulseaudio.service` |
+
+The user `pulseaudio_user` is created if not already existent.
+Also, it is added to the necessary groups for running PulseAudio.
+
+## Example Playbook
+
+```yml
+- hosts: pulseaudio
+  roles:
+    - role: pulseaudio
+      pulseaudio_user: stuvus
+```
```

Comment:
Our [Ansible role skeleton](https://github.com/stuvusIT/ansible-skeleton) adds a section for author information, did you leave that out for a reason? It's up to you but maybe you want to have the attribution.

Reply:
For project specific questions or reports we have GitHub issues.

Reply:
I asked it here because it is the initial commit of this project? Your README.md has no author information. I mentioned it because you may have forgot to add it. Feel free to merge the role if you want.